src/AspNetCore/Inc/applicationinfo.h

```diff
@@ -4,6 +4,8 @@
 #pragma once
 #define API_BUFFER_TOO_SMALL 0x80008098
 
+extern BOOL     g_fRecycleProcessCalled;
```

Comment:
We call RecycleProcess for inprocess here: https://github.com/aspnet/IISIntegration/blob/00b1948937e8d5cdcc7559c8d8aafda1613479b4/src/RequestHandler/inprocess/inprocessapplication.cpp#L172. Will there be issues if g+fRecycleProcess is false if we have called recycle already?

Reply:
I thought about it. This is impossible for inprocess. If we failed to create the recycle thread, the recycle function will not be called.